tests/training/multi_gpu_test.py

```diff
@@ -0,0 +1,68 @@
+# pylint: disable=no-self-use,invalid-name
+
+
+from copy import deepcopy
```

Comment:
Separating these imports into the three standard blocks would be nice.  And I think zero or one blank line between the top comment and the first import is enough.